src/app/inventory/item-types.ts

```diff
@@ -431,6 +431,8 @@ export interface DimSocket {
   hasRandomizedPlugItems: boolean;
   /** Is this socket a perk? Anything else is at least sorta mod-like. TODO: should this be an enum? */
   isPerk: boolean;
+  /** Is this socket reusable? This is a notably different behavior and UI in Destiny, displayed in circles rather than squares */
+  isReusable: boolean;
```

Comment:
There's a part of me that's confused now as to which one I should be looking at - isPerk or isReusable.

Reply:
we use isPerk a bunch to determine like, "importantness", i will update the comment to help distinguish

Reply:
I wonder if we're using it to generally mean "not mod" which suggests we need a "isMod"

Reply:
"mod" is the worst word in destiny, what i want is a socketType param that includes
armorMod/weaponMod (you insert a thing into this, it can be empty),
part (mag barrel etc)
perk (outlaw/rampage)
armorIntrinsic (artifice armor, exotic effects, iron lord's pride)
weaponFrame or weaponArchetype
etc etc